feathr_project/feathr/spark_provider/_synapse_submission.py

```diff
@@ -325,7 +325,7 @@ def create_spark_batch_job(self, job_name, main_file, class_name=None,
     def get_driver_log(self, job_id) -> str:
         # @see: https://docs.microsoft.com/en-us/azure/synapse-analytics/spark/connect-monitor-azure-synapse-spark-application-level-metrics
         app_id = self.get_spark_batch_job(job_id).app_id
-        url = "%s/sparkhistory/api/v1/sparkpools/%s/livyid/%s/applications/%s/driverlog/stdout/?isDownload=true" % (self._synapse_dev_url, self._spark_pool_name, job_id, app_id)
+        url = "%s/sparkhistory/api/v1/sparkpools/%s/livyid/%s/applications/%s/driverlog/stderr/?isDownload=true" % (self._synapse_dev_url, self._spark_pool_name, job_id, app_id)
```

Comment:
Is it by design to only print stderr without stdout?

Reply:
We want to get error messages here. And all error messages under stdout are also contained in stderr.

Reply:
I see, then the function name `get_driver_log` seems not accurate here. Should be `get_driver_log_err()`. But it's ok in this PR.